docs/integrations/spark.md

```diff
@@ -337,6 +334,11 @@ df.write.partitionBy("example-column").parquet(s"lakefs://${repo}/${branch}/outp
 
 The data is now created in lakeFS as new changes in your branch. You can now commit these changes or revert them.
 
+### Notes for the lakeFS Hadoop FileSystem
+
+* Since data will not be sent to the lakeFS server, using this mode maximizes data security.
+* The FileSystem implementation is tested with the latest Spark 2.X and Spark 3.X Bitnami images.
```

Comment:
Can we state that one of these is Hadoop 2 and the other with Hadoop 3?

Reply:
Done